google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/AddCloudLibrariesWizardAction.java

```diff
@@ -89,7 +110,22 @@ protected String getHelpID() {
         libraries = ImmutableList.of();
       }
 
-      this.cloudApiSelectorPanel = new GoogleCloudApiSelectorPanel(libraries);
+      this.cloudApiSelectorPanel = new GoogleCloudApiSelectorPanel(libraries, project);
+    }
+
+    /** Adds the given {@link ActionListener} to the panel's module combobox. */
+    void addModuleSelectionListener(ActionListener listener) {
+      cloudApiSelectorPanel.addModuleSelectionListener(listener);
+    }
+
+    /** Returns the selected {@link Module}. */
+    Module getSelectedModule() {
+      return cloudApiSelectorPanel.getSelectedModule();
+    }
+
+    /** Returns the set of selected {@link CloudLibrary CloudLibraries}. */
+    Set<CloudLibrary> getSelectedLibraries() {
```

Comment:
Is this idea that this wizard itself will be passed into subsequent steps?

Reply:
My thoughts are that the surrounding wizard class (`AddCloudLibrariesWizard`) is responsible for the interplay between steps. If a step requires items from a previous step, for example, it's the wizard's job to pass those along because it understands the context in which these steps are created.

So in this case, if the `ManageCloudApisStep` requires the list of selected libraries from this step, the wizard will pull the selected libraries out of here and pass them to the next step.